content/en/synthetics/private_locations/configuration.md

```diff
@@ -80,6 +80,8 @@ On browser tests, the DNS resolution is done directly by the browser, which usua
 **Default**: `none`<br>
 Proxy URL used by the private location to send requests to Datadog (for example, `--proxyDatadog=http://<YOUR_USER>:<YOUR_PWD>@<YOUR_IP>:<YOUR_PORT>`).
 
+**Note:** When setting up an HTTPS proxy, the `HTTP CONNECT` request made to the proxy establishes the initial TCP connection between the Private Location and Datadog. As such, reverse proxies like HAProxy that directs `HTTP CONNECT` request to Datadog are not supported. A forward proxy should be set up to open the connection to Datadog on behalf of the Private Location.
+
```

Comment:
Language has been adapted from the [Agent proxy configuration doc page](https://docs.datadoghq.com/agent/configuration/proxy/?tab=linux#web-proxy), but the behavior is different with the Private Location and the Synthetics intake.